packages/global/components/layouts/content/default.marko

```diff
@@ -53,6 +53,13 @@ $ const shouldInjectAds = ["article", "video", "news", "podcast"].includes(type)
         </default-theme-page-dates>
       </if>
       <global-sponsored-section-logo alias=primarySection.alias modifiers=["content-page"] class="mt-3" />
+      <if(displaySocialShare)>
```

Comment:
This should be removed (it was handled in a separate PR and doesn't relate to the inquiry button)